source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet

```diff
@@ -11,25 +11,26 @@ local question(title) = {
       },
     ],
   },
-  type: 'General',
+  type: 'MutuallyExclusive',
+  mandatory: false,
   answers: [
     {
       id: 'armed-forces-answer',
       mandatory: false,
+      type: 'Checkbox',
       guidance: {
         show_guidance: 'Why your answer is important',
         hide_guidance: 'Why your answer is important',
         contents: [
           {
-            description: 'We are measuring the number of people who have served in the UK Armed Forces and have now left. Government and councils need this information to carry out their commitments made under the Armed Forces Covenant. This is a promise by the nation ensuring that those who serve or who have served in the armed forces, and their families, are not disadvantaged.',
+            description: 'Your answer will help your local community by providing information needed to support people who used to serve in the UK Armed Forces but have left.',
```

Comment:
Not exactly as it should be: "people who used to serve in the UK Armed Forces but have now left" & "This is a promise by the nation to ensure..." below